### PR TITLE
feat(design-land): added DaffImageGallery to design land page

### PR DESCRIPTION
**Pull request by Nolan-Arnold**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Comment by Nolan-Arnold** [comment]
All images were taken from https://pixabay.com/images/search/daffodil/?pagi=2 as free for commercial use and no attribution required.

File: apps/design-land/src/app/image-gallery/image-gallery.component.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+@Component({
+  selector: 'design-land-image-gallery',
+  templateUrl: './image-gallery.component.html',
+  styleUrls: ['./image-gallery.component.scss']
+})
+export class ImageGalleryComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
```
Comment (damienwebdev) [inline apps/design-land/src/app/image-gallery/image-gallery.component.ts]: I think in the `ngOnInit` we should show an example of defaulting to a particular image when the component loads.

Comment (damienwebdev) [inline apps/design-land/src/app/image-gallery/image-gallery.component.ts]: I think in the ngOnInit we should show an example of defaulting to a particular image when the component loads

Comment (Nolan-Arnold) [reply apps/design-land/src/app/image-gallery/image-gallery.component.ts]: Okay this is taken care of in the newest commit

**Pull request merged**